client.go

```diff
@@ -429,6 +433,29 @@ func (client *client) Leader(topic string, partitionID int32) (*Broker, error) {
 	return leader, err
 }
 
+func (client *client) RefreshBrokers() {
```

Comment:
There is already RefreshController and Brokers that work on latest metadata. Can you please elaborate on what problem does it solve??

Reply:
Yes, a function already exists. But it does not work on project kafka-exporter with 'client.RefreshMetadata()': https://github.com/danielqsj/kafka_exporter/issues/134#issuecomment-624670818, so I add a
function to refresh brokers specially.